lib/nmatrix/math.rb

```diff
@@ -364,6 +364,37 @@ def cov(opts={})
     deviation_scores.transpose.dot(deviation_scores) / denominator
   end
 
+  # Raise a square matrix to a power. Be careful of buffer overflows!
```

Comment:
Colin is correct.
On Thu, Jan 29, 2015 at 9:11 PM Colin J. Fuller <notifications@github.com>
wrote:

> In lib/nmatrix/math.rb
> <https://github.com/SciRuby/nmatrix/pull/316#discussion_r23820049>:
>
> > @@ -364,6 +364,37 @@ def cov(opts={})
> >      deviation_scores.transpose.dot(deviation_scores) / denominator
> >    end
> >
> > +  # Raise a square matrix to a power. Be careful of buffer overflows!
>
> Yeah, I would have called that numeric overflow. In my head buffer
> overflow is when you've allocated a buffer of, e.g., 10 chars, and you fill
> it up and then write to the 11th and maybe you segfault. Not 100% sure if
> that's the standard definition of those terms, though?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciRuby/nmatrix/pull/316/files#r23820049>.
>